tests/Feature/EngagementTest.php

```diff
@@ -197,17 +195,15 @@
     $response->assertRedirect(localized_route('engagements.manage', $engagement));
     $response = $this->actingAs($user)->get(localized_route('engagements.manage', $engagement));
 
-    $response->assertSee('Women');
-    $response->assertSee('Non-binary people');
-    $response->assertSee('Gender non-conforming people');
-    $response->assertSee('Gender fluid people');
-    $response->assertSee('Trans people');
-    $response->assertSee('2SLGBTQIA+ people');
+    $response->assertSee(Identity::whereJsonContains('clusters', IdentityCluster::GenderAndSexuality)->first()->name);
+    foreach (Identity::whereJsonContains('clusters', IdentityCluster::GenderDiverse)->pluck('name') as $identity) {
+        $response->assertSee($identity);
+    }
 
     $data = UpdateEngagementSelectionCriteriaRequest::factory()->create([
         'intersectional' => 0,
         'other_identity_type' => 'indigenous-identity',
-        'indigenous_identities' => IndigenousIdentity::all()->pluck('id')->toArray(),
+        'indigenous_identities' => Identity::whereJsonContains('clusters', IdentityCluster::Indigenous)->pluck('id')->toArray(),
```

Comment:
For this case, can [`modelKeys`](https://laravel.com/docs/9.x/eloquent-collections#method-modelKeys) be used instead.

```suggestion
        'indigenous_identities' => Identity::whereJsonContains('clusters', IdentityCluster::Indigenous)->modelKeys(),
```